packages/lib/versionInfo.test.ts

```diff
@@ -0,0 +1,132 @@
+import versionInfo from './versionInfo';
+import { reg } from './registry';
+import { Plugins } from './services/plugins/PluginService';
+import Plugin from './services/plugins/Plugin';
+
+jest.mock('./registry');
+
+const info = jest.spyOn(console, 'info').mockImplementation(() => {});
```

Comment:
In fact I'm not sure why this is needed? In general we don't want code to print to the console, as that should only be used for debugging

Reply:
I added that  to remove console logs during the tests. There was no test for `versionInfo()` until now but as soon as I added tests, the statement in `versionInfo()` was polluting the test output.

Reply:
That makes sense, but I guess now it can be removed? Or is versionInfo() still printing something?

Reply:
Yes it can be removed. versionInfo() doesn't print anything anymore.

Reply:
Ok great, if you could remove it then and create a pull request with it that would be perfect